imagetest/test_suites/network/setup.go

```diff
@@ -10,10 +10,17 @@ import (
 var Name = "network"
 
 const (
-	vm1Name = "vm1"
-	vm2Name = "vm2"
-	vm1IP   = "192.168.0.2"
-	vm2IP   = "192.168.0.3"
+	vm1Name    = "vm1"
```

Comment:
with all these constants I wonder if wouldn't be more elegant to introduce a `type InstanceConfig struct` and wrap it all with it.

Reply:
Done.